c++11.pri

```diff
@@ -3,7 +3,8 @@ macx {
   # We attempt to auto-detect it by inspecting Boost
   dirs = $${BOOSTDIR} $${QMAKE_LIBDIR}
   for(dir, dirs) {
-    system(grep -q __112basic_string $${dir}/libboost_thread* >& /dev/null) {
+    #system(grep -q __112basic_string $${dir}/libboost_thread* >& /dev/null) {
+    system(otool -L /usr/local/lib/libboost_thread*  | grep libc++ >& /dev/null ) {
```

Comment:
This searches in `/usr/local/lib`, which is not necessarily where boost is located.